src/store/login/saga.ts

```diff
@@ -162,14 +170,18 @@ export function* openFirstConversationAfterChannelsLoaded() {
   }
 }
 
+function* listenForLogoutEvents() {
```

Comment:
Can we rename this to `listenForUserLogout` (same as used previously).

Also one question, i see that we've implemented this listener in 2 places. Just to confirm this relates to the "single responsibility principle" that you were talking about, right? Otherwise we could have just added a single listener, and if user logout is received, just do a) `stopChannelRefresh(..)`, b) `reset()` together ..?

Reply:
That's right. This one is here because it's managing the state for the Login flow. The `reset` here is based on the context of this set of sagas and store data.

Reply:
got it, thanks